test/unit/modules/transactions.js

```diff
@@ -0,0 +1,411 @@
+			[transactionTypes.SEND]: function (cb) {
+				var data = 'extra information';
+				var transferTrs = node.lisk.transaction.createTransaction(node.gAccount.address, 112340000, transferAccount.password, null, data);
+				addTransaction(transferTrs, mergeResponseAndTransaction(transferTrs, cb));
+			},
+			[transactionTypes.SIGNATURE]: function (cb) {
+				var signatureTrs = node.lisk.signature.createSignature(signatureAccount.password, signatureAccount.secondPassword);
+				addTransaction(signatureTrs, mergeResponseAndTransaction(signatureTrs, cb));
+			},
+			[transactionTypes.DELEGATE]: function (cb) {
+				var delegateTrs = node.lisk.delegate.createDelegate(delegateAccount.password, delegateAccount.username);
+				addTransaction(delegateTrs, mergeResponseAndTransaction(delegateTrs, cb));
+			},
+			[transactionTypes.VOTE]: function (cb) {
+				var votes = [
+					'+9d3058175acab969f41ad9b86f7a2926c74258670fe56b37c429c01fca9f2f0f', 
+					'+141b16ac8d5bd150f16b1caa08f689057ca4c4434445e56661831f4e671b7c0a'
+				];
+				var voteTrs = node.lisk.vote.createVote(voteAccount.password, votes);
+				addTransaction(voteTrs, mergeResponseAndTransaction(voteTrs, cb));
+			},
+			[transactionTypes.MULTI]: function (cb) {
+				var lifetime = 1;
+				var min = 2;
+				var keysgroup = [
+					'+' + multiAccount2.publicKey,
+					'+' + multiAccount3.publicKey
+				];
+
+				var multiTrs = node.lisk.multisignature.createMultisignature(multiAccount1.password, null, keysgroup, lifetime, min);
+				addTransaction(multiTrs, mergeResponseAndTransaction(multiTrs, function (err, res) {
+					var signature1 = node.lisk.multisignature.signTransaction(multiTrs, multiAccount2.password);
+					var signature2 = node.lisk.multisignature.signTransaction(multiTrs, multiAccount3.password);
+					async.each([signature1, signature2], function (signature, eachCb) {
+						postSignature(multiTrs, signature, eachCb);
+					}, function (err1) {
+						cb(err || err1, res);
+					});
+				}));
+			},
+			[transactionTypes.DAPP]: function (cb) {
+				var options = {
+					type: 1,
+					name: dappName,
+					category: 2,
+					link: 'http://www.lisk.io/' + dappName + '.zip',
+				};
+				var dappTrs = node.lisk.dapp.createDapp(dappAccount.password, null, options);
+				addTransaction(dappTrs, mergeResponseAndTransaction(dappTrs, function (err, res) {
+					node.onNewBlock(function (err1) {
+						cb(err || err1, res);
+					});
+				}));
+			}
+			/**
+			 *  Need to construct outTransfer using liskJs, commented code is for inTransfer
+				[transactionTypes.IN_TRANSFER]: [String(transactionTypes.DAPP), function (res, cb) {
+					var dappId = res[transactionTypes.DAPP].transactionId;
+					var inTransferTrs = node.lisk.transfer.createTransfer(transferAccount.password, null, dappId);
+					addTransaction(inTransferTrs, mergeResponseAndTransaction(inTransferTrs, function (err, res) {
+						node.onNewBlock(function (err1) {
+							cb(err, res);
+						});
+					}));
+				}]
+			**/
```

Comment:
`inTransfer` and `outTransfer` construction is fixed in `lisk-js` from `0.4.4`, or should be.